build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -15,7 +15,8 @@
       <RanLib>$(_ArmRanLib)</RanLib>
       <Strip>$(_ArmStrip)</Strip>
       <ConfigureFlags>--host=armv5-linux-androideabi $(_TargetConfigureFlags)</ConfigureFlags>
-      <OutputRuntime>libmonosgen-2.0.so</OutputRuntime>
+      <OutputRuntime>libmonosgen-2.0</OutputRuntime>
+      <OutputRuntimeExtension>so</OutputRuntimeExtension>
```

Comment:
Indeed, I like that. However, we don't need `%(OutputRuntimeExtension)`; the extension will be the same for all those `%(Output*)` properties. Thus, `%(OutputRuntimeFilename)` and `%(NativeLibraryExtension)`?